e2e/example.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
```

Comment:
That's an extraneous dependency and the eslint throws an error on dev mode, it doesn't affect anything to this code, but to keep the culture of an error-free environment we should add a `// eslint-disable-next-line import/no-extraneous-dependencies` to this file and on `playwright.config` as well